packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -257,7 +257,7 @@ export async function load_node({
 								fetched.push({
 									url: requested,
 									body: /** @type {string} */ (opts.body),
-									json: `{"status":${response.status},"statusText":${s(response.statusText)},"headers":${s(headers)},"body":"${escape_json_value_in_html(body)}"}`
+									json: `{"status":${response.status},"statusText":${s(response.statusText)},"headers":${s(headers)},"body":${escape_json_in_html(body)}}`
```

Comment:
just to be safe I think we should double check that the `status` is a number. I have no idea whether that would already be validated on all platforms
```suggestion
									json: `{"status":${Number(response.status)},"statusText":${s(response.statusText)},"headers":${s(headers)},"body":${escape_json_in_html(body)}}`
```

Reply:
this would result in `"status": NaN` for non numberish which would then result in parse errors down the road.
Should we validate that response.status is an actual integer in the valid http status range here or is it done elsewhere already?

Reply:
a check could look like this

```js
const statusNumber = Number(response.status);
if (isNaN(statusNumber)) {
 throw new Error(`response.status is not a number. value: "${response.status}" type: ${typeof response.status}`);
}
```
and then use statusNumber in the json string.

Reply:
that sounds fine. whether it's worth adding the extra validation to throw an error or not is something I debated. I think either way would be fine. I just want to be sure we're not inadvertently injecting something from the user into the page without any validation or escaping with the `Number` alone would do

Reply:
i think throwing here is better, the subsequent parse error would be less tracable. added